pylint/checkers/stdlib.py

```diff
@@ -463,6 +463,13 @@ class StdlibChecker(DeprecatedMixin, BaseChecker):
                 ]
             },
         ),
+        "W1519": (
+            "Using next without explicitly specifying a default value or catching the StopIteration",
```

Comment:
```suggestion
            "Using next without specifying a default value or catching the StopIteration",
```
There's no implicit default, it's going to raise a StopIteration if you don't set the default right ?